analyzer/tests/functional/ctu/test_ctu.py

```diff
@@ -58,21 +61,36 @@ def setUp(self):
         print("'analyze' reported CTU-on-demand-compatibility? " +
               str(getattr(self, ON_DEMAND_ATTR)))
 
+        self.buildlog = os.path.join(self.test_workspace, 'buildlog.json')
+        self.complex_buildlog = os.path.join(
+            self.test_workspace, 'complex_buildlog.json')
+
         # Fix the "template" build JSONs to contain a proper directory
         # so the tests work.
-        raw_buildlog = os.path.join(self.test_dir, 'buildlog.json')
-        with open(raw_buildlog,
-                  encoding="utf-8", errors="ignore") as log_file:
-            build_json = json.load(log_file)
+        def adjust_buildlog(buildlog_filename: str):
```

Comment:
I think this `adjust_buildlog` method could be useful for other tests too, or is this adjustment really specific for this test? Only the source file paths are updated.
There are already some helpers for the test environment here `codechecker/analyzer/tests/libtest/`

Reply:
Good Point!
I have put the utility into `codechecker/analyzer/tests/libtest/env.py`, it should be usable by other tests now. Another round will be necessary to check where this can be used.